.github/workflows/test.yml

```diff
@@ -0,0 +1,65 @@
+name: test
+
+on: [pull_request]
```

Comment:
Can we configure it to run periodically (weekly?) on master just to double check that master hasn't broken?

Reply:
Yes that's possible: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule though I'm curious as to how it would break spontaneously?

Reply:
Actually I suppose stuff like the API used in the tutorial might break

Reply:
Yes we can, though it becomes a bit of a tree falling in the forrest problem. If the action fails but no-one bothers to look, did it fail?

This issue applies to all of our repos (due to inter-dependence) so I think we will require a more integrated solution, there is a an issue open somewhere (I think on cylc admin) to setup nightly builds to catch these sorts of problems.

Reply:
Let's not worry about this then...